docs/databags.md

```diff
@@ -58,7 +62,7 @@ An Indexes Hash is part of a plaintext data bag item that defines the set of ind
 
 Roles Hash
 ----------
-A Roles Hash is a contextual (see above) Hash, part of a plaintext data bag item that defines roles for every node in a cluster, and is pointed to by the `node[:splunk][:config][:roles]` attribute (usually set in your environment).
+A Roles Hash is a contextual (see above) Hash, part of a plaintext data bag item that defines roles for every node in a cluster, and is pointed to by the `node[:splunk][:config][:roles]` attribute (usually set in your environment). A special key of 'shc_configs' is used for managing the roles on the search heads in a search head cluster.
```

Comment:
I don't have a strong preference either way. deployer-apps seems consistent with master-apps in that it's named for the node that chef will deploy the apps to.